Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX_test.py

```diff
@@ -120,6 +121,51 @@ def json(self):
     "filter": "",
 }
 
+def test_running_polling_command_upload_file_error(mocker):
+    """
+    Given
+    - A client object
+    - IDs of files that have been uploaded as part of the polling command
+    When
+    - Running the 'cs-fx-upload-file' command with polling
+    Then
+    - Validate that we return an error when the report contains an error object, meaning
+    that the sandbox analysis was not able to run properly
+    """
+    args = {'ids': '1234', 'extended_data': 'true'}
+    mocker.patch.object(Client, '_get_access_token')
+    client = Client(server_url="https://api.crowdstrike.com/", username="user1", password="12345", use_ssl=False,
+                    proxy=False, reliability=DBotScoreReliability.B)
+    mocker.patch.object(Client, 'upload_file', return_value=UPLOAD_FILE_HTTP_RESPONSE)
+    mocker.patch.object(Client, 'send_uploaded_file_to_sandbox_analysis', return_value=SEND_URL_TO_SANDBOX_ANALYSIS_HTTP_RESPONSE)
+    mocker.patch.object(Client, 'get_full_report', return_value=GET_FULL_REPORT_HTTP_RESPONSE_ERROR_MESSAGE)
+    with pytest.raises(DemistoException) as e:
+        run_polling_command(client, args, 'cs-fx-upload-file', upload_file_command,
+                                          get_full_report_command, 'FILE', post_function=validate_sandbox_report)
+    assert 'Sandbox was not able to analyze one of the files, failing with error' in str(e)
+
+
+@pytest.mark.parametrize('command', [(get_full_report_command), (get_report_summary_command)])
```

Comment:
Im not a big fan of using the parametrize to distinct 2 running function, I think a class is better here in general. Since the code is really the same in this case Im ok with it here but it is not the best practice to do so,